content/docs/04-clusters/03-edge/04-edgeforge-workflow/12-build-artifacts.md

```diff
@@ -0,0 +1,309 @@
+<br />
+
+  ```yaml
+  pack:
+    content:
+      images:
+        - image: "{{.spectro.pack.edge-native-byoi.options.system.uri}}"
+  options:
+    system.uri: "{{ .spectro.pack.edge-native-byoi.options.system.registry }}/{{ .spectro.pack.edge-native-byoi.options.system.repo }}:{{ .spectro.pack.edge-native-byoi.options.system.k8sDistribution }}-{{ .spectro.system.kubernetes.version }}-{{ .spectro.pack.edge-native-byoi.options.system.peVersion }}-{{ .spectro.pack.edge-native-byoi.options.system.customTag }}"
+    system.registry: ttl.sh
+    system.repo: ubuntu
+    system.k8sDistribution: k3s
+    system.osName: ubuntu
+    system.peVersion: v3.4.3
+    system.customTag: demo
+    system.osVersion: 22
+  ``` 
+  <br />
+
+  <InfoBox>
+
+  The BYOOS pack's `system.uri` attribute references the Kubernetes version selected in the cluster profile by using the `{{ .spectro.system.kubernetes.version }}` [macro](/clusters/cluster-management/macros). This is how the provider images you created and pushed to a registry are tied to the OS and Kubernetes version you selected in the **.arg** file.
+
+  </InfoBox>
+ 
+  The screenshot below displays how to reference a provider image in the BYOOS pack of your cluster profile.
+  <br />
+
+  ![A screenshot of k3s OS layer in a cluster profile.](/tutorials/edge/clusters_edge_deploy-cluster_edit-profile.png)
+
+  <WarningBox>
+
+  The ttl.sh is a short-lived image registry. If you do not use the provider image in your cluster profile within 24 hours of pushing to ttl.sh, it will no longer exist and must be re-pushed. In a production environment, use a custom registry to host provider images.
+
+  </WarningBox>
+  <br />
+
+
+9. Add the following Kubernetes layer to your cluster profile. Select the K3s version 1.25.x because earlier in this how-to guide, you pushed a provider image compatible with K3s v1.25.2 to the ttl.sh image registry. 
+
+  |**Pack Type**|**Registry**|**Pack Name**|**Pack Version**| 
+  |---|---|---|---|
+  |Kubernetes|Public Repo|Palette Optimized K3s|`1.25.x`|
+
+
+10. Add the network layer to your cluster profile, and choose a Container Network Interface (CNI) pack that best fits your needs, such as Calico, Flannel, Cilium, or Custom CNI. For example, you can add the following network layer. This step completes the core infrastructure layers in the cluster profile.  
+
+  |**Pack Type**|**Registry**|**Pack Name**|**Pack Version**| 
+  |---|---|---|---|
+  |Network|Public Repo|Calico|`3.25.x`|
+    
+  
+11. Add add-on layers and manifests to your cluster profile per your requirements. 
+
+
+12. If there are no errors or compatibility issues, Palette displays the newly created complete cluster profile for review. Verify the layers you added, and finish creating the cluster profile. 
+
+
+13. The remaining steps will guide you to create an offline content bundle using the Palette Edge CLI and build the Edge installer ISO. 
```

Comment:
Step 13 should be a prereq, at least the steps on how to create the API key. If anything, we need to improve the steps [here](https://docs.spectrocloud.com/user-management/user-authentication/#creatinganapikeyasatenantadmin) and just link to this section.